packages/docmanager-extension/src/index.ts

```diff
@@ -746,6 +746,12 @@ function addCommands(
     selector: '[data-type="document-title"]',
     rank: 5
   });
+  // .jp-mod-current added to open console from a right click on a tab for the selected (blue line) txt file or notebook
```

Comment:
```suggestion
  // .jp-mod-current added so that the console-creation command is only shown on the current document.
  // Otherwise it will delegate to the wrong widget.
```